src/state/pools/index.ts

```diff
@@ -151,7 +151,7 @@ export const updateUserPendingReward =
     dispatch(updatePoolsUserData({ sousId, field: 'pendingReward', value: pendingRewards[sousId] }))
   }
 
-export const fetchCakeVaultPublicData = createAsyncThunk<CakeVault>('cakeVault/fetchPublicData', async () => {
+export const fetchCakeVaultPublicData = createAsyncThunk<CakeVault>('cakeVault/fetchCurrentLotteryId', async () => {
```

Comment:
em... auto-replaced by accident? 🤔

Reply:
Lol great spot. There's a function in the lottery `fetchPublicData`, will have auto-replaced. thanks!